qa/L0_perf_client/test.sh

```diff
@@ -96,6 +96,36 @@ for MODEL in graphdef_nobatch_int32_int32_int32 graphdef_int32_int32_int32; do
     done
 done
 
+# Testing with the new arguments
+set +e
+$PERF_CLIENT -v -i grpc -u localhost:8001 -m graphdef_int32_int32_int32 >$CLIENT_LOG 2>&1
```

Comment:
We should make the default -u value be based on the -i. So if -u is not specified explicitly the default for http should be localhost:8000 and the default for grpc should be localhost:8001